test/cli/exec/run

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -eux
+
+absolute_path() {
+  echo "$(cd "$(dirname "$1")" && pwd)/$(basename "$1")"
+}
+
+MAFIA=$(absolute_path ${1:-./dist/build/mafia/mafia})
+
+MAFIA_TEMP=$(mktemp -d 2>/dev/null || mktemp -d -t 'mafia')
+trap "rm -rf \"${MAFIA_TEMP}\"" EXIT
+
+cd ${MAFIA_TEMP}
+
+MAFIA_HOME=${MAFIA_TEMP} ${MAFIA} exec -- ghc-pkg list base
```

Comment:
How do you know this is working? `ghc-pkg list base` would also succeed if you just ran it from outside a sandbox